tools/test_deleted_renamed_testdata

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
+    if [ -f "$FILE" ]; then
+        # if file exists it means it was renamed, and then original file name is retrieved by git log
+        FILE=$(git log --follow -p $FILE | grep 'rename from test_data/')
+    fi
+    if MATCHED_SCHEDULE_FILES="$(grep --recursive --ignore-case --files-with-matches "$FILE\b" schedule/)"
```

Comment:
did you consider to use the same script for test_data and schedule files, it looks like only [these two lines](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tools/test_deleted_renamed_referenced_modules#L11-L12) are different which could be parameters, wdyt?

Reply:
hmm.. How could the input be correct , since the command in .PHONY is greping files in different locations?

Reply:
you're right, I found difficulties to store the array and call `git diff` from the script and store the array in borne shell, if you feel fancy you can try out, otherwise it is fine as it is.

Reply:
Another option (just a suggestion), instead of doing the call from the script, we could pass the flag and use shift to get the flag inside the script and the rest would remain as it is. Check https://www.tldp.org/LDP/Bash-Beginners-Guide/html/sect_09_07.html but I never used that, Idk how tricky could be.

Reply:
I investigated that a bit. I think "tools/test_deleted_renamed_testdata" can be deleted and then we can modify tools/test_deleted_renamed_referenced_modules with something like :

>FILES="${@}"
>success=1
>USAGE1="Usage : $filetype pm yaml"
>for i in 1 2
>do
>       filtered_files=grep $filetype $FILES
>for FILE in $filtered_files; do
>    if [ -f "$FILE" ]; then
>        # if file exists it means it was renamed, and then original file name is retrieved by git log
>        FILE=$(git log --follow -p $FILE | grep 'rename from test')
>   fi
>    # module name as appears in scheduling files excluding 'tests/' and file extension
>    module=$(echo $FILE | cut -f 2- -d '/' | cut -f 1 -d '.')
>   target_paths='schedule/ products/*/main.pm lib/main_common.pm'
>    USAGE2="Usage : $target_paths schedule/"
>   USAGE3="Usage : $filename $module $FILE"
>   if MATCHED_SCHEDULE_FILES="$(grep --recursive --ignore-case --files-with-matches "${module}\b" $target_paths)"
>    then
>       echo -e "\"$module\" test module was removed or renamed, but it is still used in: \
        \n$MATCHED_SCHEDULE_FILES\n"
>        success=0
 >   fi
>done
>shift
>done
>[ $success = 1 ] && echo "SUCCESS" && exit 0
>exit 1

But that looks like it increases complexity and I don't think we are saving time or anything. Just an extra file.

Reply:
I see, interesting solution, but I agree with you, having loops in the script increase complexity for debuging, ideal IMO is to call the script twice with different parameters than just the array. But let's leave it as it is, it is fine to have two files.